src/extension.ts

```diff
@@ -697,40 +691,34 @@ export class Ext extends Ecs.System<ExtEvent> {
             this.last_focused = win.entity;
         }
 
-        function activate_in_stack(ext: Ext, stack: node.NodeStack, win: Window.ShellWindow) {
-            ext.auto_tiler?.forest.stacks.get(stack.idx)?.activate(win.entity);
-        }
-
-        if (this.auto_tiler) {
+        if (null !== this.auto_tiler) {
             win.meta.raise();
 
             // Update the active tab in the stack.
-            const attached = this.auto_tiler.attached.get(win.entity);
-            if (attached) {
-                const fork = this.auto_tiler.forest.forks.get(attached);
-                if (fork) {
-                    if (fork.left.is_in_stack(win.entity)) {
-                        activate_in_stack(this, (fork.left.inner as node.NodeStack), win);
-                    } else if (fork.right?.is_in_stack(win.entity)) {
-                        activate_in_stack(this, (fork.right.inner as node.NodeStack), win);
-                    }
-                }
+            if (null !== win.stack) {
+                ext?.auto_tiler?.forest.stacks.get(win.stack)?.activate(win.entity)
             }
         }
 
         this.show_border_on_focused();
 
-        if (this.auto_tiler && this.prev_focused !== null && win.is_tilable(this)) {
+        if (this.auto_tiler && this.prev_focused !== null) {
```

Comment:
This line is responsible for https://github.com/pop-os/shell/issues/942, moving the mouse in VLC displays the full-screen controller windows that trigger `unmake_fullscreen()` whitout the `win.is_tilable(this)` check.